### DISCUSSION TITLE
Saleor dashboard logging me out so quickly after very few mins

### DISCUSSION THREAD

**Discussion by amineff** [saleor/saleor#9956] [General]
I am using the latest version of saleor dashboard and back-end (3.4.3 - 3.4.0) and the dash board is logging me out so quickly, is there anyway to prevent token expiration and save my login ?

**Comment by timuric** [comment]
@amineff are you using Cloud version or running it locally?

**Reply by barrybhattarai** [reply]
What is the default for JWT_TTL_REFRESH in the cloud version? Thank you in advance:)

**Comment by demodes** [comment]
Maybe setting this would help:

JWT_TTL_ACCESS
The time until JWT access tokens expire. The value should be a time expressions like 5m, 5 minutes, 5d, 5 days, 1w, 1 week. Defaults to 5 minutes.

JWT_TTL_REFRESH
The time until JWT refresh tokens expire. The value should be a time expressions like 5m, 5 minutes, 5d, 5 days, 1w, 1 week. Defaults to 30 days.

https://docs.saleor.io/docs/3.x/developer/running-saleor/configuration
